lib/gui/app.js

```diff
@@ -79,17 +79,24 @@ app.run(() => {
     '\\____/ \\__\\___|_| |_|\\___|_|',
     '',
     'Interested in joining the Etcher team?',
-    'Drop us a line at join+etcher@resin.io'
+    'Drop us a line at join+etcher@resin.io',
+    '',
+    `Version = ${packageJSON.version}, Type = ${packageJSON.packageType}`
```

Comment:
Is it worth including the detected OS here too? \*shrug\*
(at the moment each unique `packageType` only maps to a single OS, but I guess that's not guaranteed)